app/src/main/kotlin/io/element/android/x/MainActivity.kt

```diff
@@ -98,26 +99,38 @@ class MainActivity : NodeActivity() {
                         .fillMaxSize()
                         .background(MaterialTheme.colorScheme.background),
                 ) {
-                    NodeHost(integrationPoint = appyxIntegrationPoint) {
-                        MainNode(
-                            it,
-                            plugins = listOf(
-                                object : NodeReadyObserver<MainNode> {
-                                    override fun init(node: MainNode) {
-                                        Timber.tag(loggerTag.value).w("onMainNodeInit")
-                                        mainNode = node
-                                        mainNode.handleIntent(intent)
-                                    }
-                                }
-                            ),
-                            context = applicationContext
```

Comment:
Not ideal to touch MainActivity, but I want the migration code to be run without using Appyx.